packages/next/build/index.ts

```diff
@@ -1527,7 +1525,7 @@ export default async function build(
 
     const images = { ...config.images }
     const { deviceSizes, imageSizes } = images
-    images.sizes = [...deviceSizes, ...imageSizes]
+    ;(images as any).sizes = [...deviceSizes, ...imageSizes]
```

Comment:
Should this be `images!.sizes`? I think we should avoid `any` if we can?

Reply:
`sizes` isn't a [config value](https://github.com/vercel/next.js/blob/f52955ec94e6d1505792b02d88048e18999af5be/packages/next/server/image-config.ts#L11), we're changing the shape to the manifest `images` type instead here it looks like, we could create a completely separate object below instead

Reply:
Oh yeah, this was because we had a early canary that was using `sizes` and then split it to `imageSizes` and `deviceSizes`. But since the backend already used `sizes` this was kept for backwards compat 👍